esp/src/eclwatch/ESPUtil.js

```diff
@@ -242,6 +243,15 @@ define([
         }
     });
 
+    var SessionWatcher = dojo.declare([Evented], {
+        unlocked: function(){
```

Comment:
This confused me - I think it would be better if it was called "monitorLockClick"?